### ISSUE TITLE
Track each time an issue pack is used

### ISSUE THREAD

**Issue by cmbirk** [GovReady/Issue-Packs-Site#26]
Which issue pack, and what repo it was synced to.  This is dependent on #24

**Comment by cmbirk** [comment]
This is currently working on user uploaded packs.  It tracks the user id, pack id, repo name, and url.  I'm still working out a way to track syncs for our "Official Packs", as they are pulled from our GH repo and don't have IDs.

Here's a couple options I'm thinking through.  I very strongly recommend the first one.  @gregelin thoughts?

* Users can't sync official packs, but they can copy them to their own pack section.  They could then sync / track, and it would negate any concerns about official packs changing, as they'd have a copy of the version they want to use.  We would just present the "Official Packs" a bit differently so as not to confuse them as installable and make the copy / sync workflow very clear.

* We create an Admin layer that allows us to store 'Admin Packs' in the database.  In the 'Admin Dashboard', there would be an easy way to pull from our YAML repo into the application database.  This is more effort because we have to create the 'Admin' authentication layer and a separate Admin dashboard view.

* We hard-code a user email into the system that acts as the "Official GovReady Account".  For example, we could hard-code my email into the system and any packs I upload are seen as "Official Packs".  Users would still have to copy my packs into their account for this option.

* Upon attempting to copy an "Official Pack", a copy of that pack is auto-copied into the user's packs and synced from there.  I think the UI / notifications get messy with this one, and we don't want to surprise the user.

**Comment by gregelin** [comment]
@cmbirk, It would make sense to implement users copying or "cloning" a pack into their own packs. Let's start their.

It also makes sense to have an account for official GovReady packets. That can be hard-coded or just added to database as govready. The account is best associated with my email, or "ops@govready.com."

Thanks.

**Comment by gregelin** [comment]
@cmbirk Why can't we add ids to the "official" Issue packs in the repo?

**Comment by cmbirk** [comment]
The ids are auto-incrementing columns in the issuepack table in the MySQL database.  We're pulling the official packs from the YAML files in the repo, so there's no way to assign relevant ids.

**Comment by cmbirk** [comment]
These are being tracked, so I'm closing this in favor of #29
